frontend/src/v5/ui/components/shared/revisionDetails/components/revisionsListItemButton/revisionsListItemButton.styles.ts

```diff
@@ -19,6 +19,8 @@ import { Button as ButtonComponent } from '@controls/button';
 
 export const Container = styled.div`
 	min-width: 80px;
+	position: absolute;
+    right: 60px;
```

Comment:
![image](https://user-images.githubusercontent.com/62481250/184118609-a42fc4dd-48ff-4d3c-9774-a6558a31da0a.png)

Reply:
I need it to be aligned to the right. Any suggestions there?